src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -172,11 +172,21 @@ def login(cmd, username=None, password=None, service_principal=None, tenant=None
                                # pylint: disable=line-too-long
                                "More details are available at https://github.com/AzureAD/microsoft-authentication-library-for-python/wiki/Username-Password-Authentication")
         raise CLIError(err)
-    except requests.exceptions.SSLError as err:
-        from azure.cli.core.util import SSLERROR_TEMPLATE
-        raise CLIError(SSLERROR_TEMPLATE.format(str(err)))
+    except requests.exceptions.SSLError:
+        from azure.cli.core.azclierror import UnclassifiedUserFault
+        raise UnclassifiedUserFault(error_msg='Certificate verification failed.'
+                                    ' This typically happens when using Azure CLI behind a proxy '
```

Comment:
A white space usually goes to the previous line.

Reply:
done.